prime-angular-frontend/src/app/shared/components/forms/college-certification-form/college-certification-form.component.ts

```diff
@@ -68,7 +68,8 @@ export class CollegeCertificationFormComponent implements OnInit {
   ) {
     this.remove = new EventEmitter<number>();
     // filter the college licenses that have been discontinued
-    this.licenses = this.configService.licenses.map(l => {
+    this.licenses = this.configService.licenses.map(x => Object.assign({}, x));
```

Comment:
Please add comment re: cloning

Reply:
Can you also put this comment "Each license object should have at least one collegeLicense object in the master list (configService.license) so that other code can identify either the collegeLicense is discontinued or not." somewhere, perhaps in `config.service`, around line 52?

Reply:
The comment is updated with more detail.